CONTRIBUTING.md

```diff
@@ -222,28 +222,31 @@ In the case of testing the `API`, we would like to split all tests into `mutatio
 To run tests, enter `poe test` in your terminal.
 
 ```bash
-poe test
+uv run poe test
 ```
 
 By default `poe test` is using the `--reuse-db` flag to speed up testing time.
 
 > [!TIP]
 > If you need to ignore `--reuse-db` (e.g when testing Saleor on different versions that have different migrations) add `--create-db` argument: `poe test --create-db`
 
+> [!TIP]
+> Depending on your setup, you may want to run your Postgres on host (not inside container) to improve the test execution speed
+
```

Comment:
Not sure why this change is marked as new, it already exists on main 🤷‍♀️

Reply:
What do you mean by `marked as new`? It looks like ` tip section for setup 🤔